lib/yt/actions/base.rb

```diff
@@ -29,4 +29,4 @@ def sanitize_brackets!(source)
       end
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
You are right that there should be a new line here. 

However… either make it a separate commit or don't add it here.

It's just confusing to see this file touched as part of this PR.

Thanks.